### PR TITLE
Adding detector and recognizer modules with corresponding unittest

### PR DESCRIPTION
**Pull request by hungpham3112**
NEED REVIEW BEFORE MERGE TO DEV BRANCH

### CODE REVIEW

File: model/test_detector.py
Comment (Huy1711) [inline model/test_detector.py:1]: Please put test file inside `/test` folder

File: model/detector.py
Comment (Huy1711) [inline model/detector.py:1]: please put this file inside src/model folder

File: model/__init__.py
Comment (Huy1711) [inline model/__init__.py:1]: please put this file inside src/model folder

File: model/recognizer.py
Comment (Huy1711) [inline model/recognizer.py:1]: please put this file inside src/model folder

File: model/test_recognizer.py
Comment (Huy1711) [inline model/test_recognizer.py:1]: please put this file inside test/ folder

File: model/detector.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import cv2
+import numpy as np
+from PIL import Image
+
+class TextDetector:
+    def resize_image(self, image: Image.Image, max_size: tuple = (2000, 2000)) -> tuple:
```
Comment (Huy1711) [inline model/detector.py:7]: You can use return type hint like this:

from typing import Tuple

def resize_image(...) -> Tuple[Image, float]:

File: model/recognizer.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import pytesseract
+import numpy as np
+from typing import List, Dict
+
+class TextRecognizer:
+    def __init__(self):
```
Comment (Huy1711) [inline model/recognizer.py:8]: you can delete this function if you don't implement it

**Review by Huy1711 — CHANGES_REQUESTED** [review]
Good code, but please take a look at my comments before merge

**Review by micache — APPROVED** [review]
LGTM

**Pull request closed without merging**